src/Hl7.Fhir.Core.Tests/Introspection/FhirClassMappingTests.cs

```diff
@@ -34,5 +35,34 @@ bool isNested(Type testee)
                 return cm.IsNestedType;
             }
         }
+
+        [TestMethod]
+        public void TestPerformanceOfMappingAllClasses()
+        {
+            // just a random list of POCO types available in common
+            var typesToTest = ModelInfo.FhirCsTypeToString.Keys;
+            Console.WriteLine($"Creating a mapping for all {typesToTest.Count} pocos.");
+
+            var sw = new Stopwatch();
+            sw.Start();
+            for (int i = 0; i < 10; i++)
```

Comment:
This was in a previous PR changed to `for (int i = 0; i < numRepeats; i++)`. Now it changed back to `< 10`.

Reply:
This PR was based against dstu-3, not against the PR where I corrected it.